wiki/Gameplay/Full_combo/en.md

```diff
@@ -6,10 +6,21 @@ tags:
 
 # Full combo
 
-**Full combo** (or *FC* for short) is a term used to describe a player obtaining the maximum possible [combo](/wiki/Beatmapping/Combo) on a [beatmap](/wiki/Beatmap). Full combos are scored by passing a beatmap with no misses<!-- TODO: link -->, [sliderbreaks](/wiki/Gameplay/Judgement/Slider_break), or dropped [slider ends](/wiki/Gameplay/Hit_object/Slider/Slidertail).
+**Full combo** (or *FC* for short) is a term used to describe a player passing a [beatmap](/wiki/Beatmap) without ever combo breaking. Full combos are scored by passing a beatmap with no misses<!-- TODO: link --> or [sliderbreaks](/wiki/Gameplay/Judgement/Slider_break).
 
 Scores that lost combo only via dropped slider ends are widely considered by the community to be full combos. This differs from the game client and website's display.
 
 ![Screenshot of full combo and broken combo scores](img/combo-comparison.png "The top score is a full combo, and the bottom score is a broken combo.")
 
 Because the [score](/wiki/Gameplay/Score) of an individual object in [osu!](/wiki/Game_mode/osu!) and [osu!catch](/wiki/Game_mode/osu!catch) heavily depends on combo multiplier and is virtually unbounded, full combos typically award the most score in these game modes.
+
+# Perfect full combo
+**Perfect full combo** (or *PFC* for short) is a term used to describe the game acknowledging a full combo ocurred by coloring the combo green. This is crucial for Hush-Hush medals, as whenever they are checking for the full combo conditional directly, it is referring to a perfect full combo. In all 4 game modes, Standard, Taiko, Catch, and Mania, the way this is technically processed varies:
```

Comment:
In Standard, it is known that a PFC is required to get any full combo skill medals.
In Catch, it is known that a PFC (which is identical to a FC for only this mode) is required to get any full combo skill medals.

Specifically Taiko and Mania are what use different code for "FCs".

The best example for Taiko is the typical map players use to go for 8* FC. Here is a direct score that is not a perfect full combo that gives the _The Drummer's Throne_ 8* Taiko full combo medal: https://osu.ppy.sh/scores/taiko/175971328

The best example for Mania is literally any ranked map you click on. I already said how a PFC works in mania in regard to getting a Hush-Hush medal like Mirage. As for how to get a full combo skill medal in mania, it as I said (you cannot get any misses or 50s, but you are allowed to get 100s): https://osu.ppy.sh/scores/mania/459029042

We understand that this is very unintuitive since hitting every single note and not combo breaking should be a full combo, but in the eyes of the game, that is not a perfect full combo in mania.

Reply:
I believe hush hush medals are valid regardless of a 50 or 100.

Reply:
Also, I do want to point out that sliderbreaking resulting in a 200 (mania) will still count as a "PFC". The reason I point this out not only has a reference but to infer that mania seems to only checking if no bads were hit.

Reply:
> I believe hush hush medals in the case of mania are valid regardless of a 50 or 100 (including full combo medals)

For mania full combo skill medals, no misses or 50s can be gotten.

Reply:
> I believe hush hush medals in the case of mania are valid regardless of a 50 or 100 (including full combo medals)

For mania full combo skill medals, no misses or 50s can be gotten. For Hush-Hush medals that involve the score being labeled as full combo with green, no misses, 50s, or 100s can be gotten.

> Also, I do want to point out that sliderbreaking resulting in a 200 (mania) will still count as a "PFC" in stable (and also apiv2). The reason I point this out not only has a reference but to infer that mania seems to only checking if no bads were hit (which seems to be true).

In that case, mania is even more flawed. That would mean it checks PFC purely from there being no misses, no 50s, or no 100s.